### PR TITLE
Add prune cronjob

### PR DESCRIPTION
**Pull request by angelbarrera92**
Hi!

This PR adds a new command to the CLI to prune old checkresults. It will run in form of a cronjob as any other Check :)
So we can even control/monitor how is running the pruner :)

The defaults:

`keep: 24h`

Feedback is welcomed!

### CODE REVIEW

File: internal/prune/prune.go [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+		return processCompletedResult(ctx, clientSet, result, targetDate)
+	}
+
+	log.WithField("checkresult", fmt.Sprintf("%v/%v", result.Namespace, result.Name)).
+		Debug("nothing to do with the checkresult. Not completed")
+
+	return nil
+}
+
+//nolint:lll // we need this method signature length
```
Comment (nandajavarma) [inline internal/prune/prune.go:76]: Just wondering why you choose to ignore the linting errors compared to moving the function parameters to the next line.

Comment (angelbarrera92) [reply internal/prune/prune.go:76]: Fair question. Just i don't like having the method signature in a multiline fashion. maybe im just old :(

Comment (angelbarrera92) [reply internal/prune/prune.go:76]: Let me fix them

**Review by nandajavarma — APPROVED** [review]
code generally looks good to me!

**Pull request merged**